/*Owner & Copyrights: Vance King Saxbe. A.*/<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExploreSessionClass</class>
 <widget class="QDialog" name="ExploreSessionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExploreSession</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>New Session</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtEventNo"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtEventName"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblEventDescription">
            <property name="text">
             <string>Event Description:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QTextEdit" name="txtEventDescription"/>
          </item>
          <item row="4" column="0" colspan="5">
           <widget class="QTableWidget" name="tblNewSession">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ContiguousSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>User</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Location</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Room</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>DateTime</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblEventNo">
            <property name="text">
             <string>Event No:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblEventName">
            <property name="text">
             <string>Event Name :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QComboBox" name="cmbEventType"/>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="lblEventType">
            <property name="text">
             <string>EventType :</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QPushButton" name="btnStart">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnCancel">
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="btnAutoFill">
            <property name="text">
             <string>AutoFill</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Resume Session</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="1">
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="1" column="1">
           <widget class="QComboBox" name="cmbLocation"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblLocation">
            <property name="text">
             <string>Location</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="cmbRoom"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="lblRoom">
            <property name="text">
             <string>Room</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QComboBox" name="cmbSearchBox">
            <item>
             <property name="text">
              <string>All cases</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Recent Cases</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLineEdit" name="txtSearch"/>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tblResumeSession">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ContiguousSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Event No</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Title</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Description</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Created Date</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>EventType</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Status</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFrame" name="frame_6">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="3">
           <widget class="QPushButton" name="btnResume">
            <property name="text">
             <string>Resume</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="btnViewNotes">
            <property name="text">
             <string>ViewNotes</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnClose">
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnOpen">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ExploreSessionClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
/*email to provide support at vancekingsaxbe@powerdominionenterprise.com, businessaffairs@powerdominionenterprise.com, For donations please write to fundraising@powerdominionenterprise.com*/